app/portainer/hostmanagement/open-amt/model.ts

```diff
@@ -1,41 +1,40 @@
-export interface AMTConfiguration {
-    enableOpenAMT: boolean;
-    mpsServer: string;
-    mpsUser: string;
-    mpsPassword: string;
-    certFileText: string;
-    certPassword: string;
-    domainName: string;
+export interface OpenAMTConfiguration {
+  Enabled: boolean,
```

Comment:
same, use semi-colons.

also, I guess this comes from the server that's why it's in PascalCase. We have a tech debt ticket to convert those to small case camelCase. anychance you change the server to return camelCase props? using `json:'enabled'`